arkane/statmech.py

```diff
@@ -1305,6 +1305,26 @@ def __init__(self,name,torsigma1,torsigma2,calcPath,symmetry='none',pivots1=None
        except OSError:
            pass
 
+   def getTorsions(self):
+       """
+       determine torsions, not entirely necessary for 2D-NS, but important for E2DT
```

Comment:
It's a more advanced torsion calcluation Q2DTor can do...is really designed to do that accounts for both a 2D torsion and how that torision interacts with the frequencies.  While better it isn't as useful to us because it's primarily for systems with exactly two torisions that couple with each other.